ffs/internal/cmdstorage/registry/registry.go

```diff
@@ -18,30 +18,15 @@
 package registry
 
 import (
-	"archive/zip"
-	"context"
-	"strings"
-
-	"github.com/creachadair/ffs/blob"
 	"github.com/creachadair/ffs/blob/memstore"
 	"github.com/creachadair/ffs/storage/filestore"
-	"github.com/creachadair/ffs/storage/zipstore"
 	"github.com/creachadair/ffstools/lib/store"
 )
 
 // Stores enumerates the storage implementations are built in by default.
```

Comment:
Missing word: "the storage implementations **that** are built in by default"

Reply:
Fixed, thanks.